Cartfile

```diff
@@ -1,4 +1,4 @@
-#github "readium/r2-shared-swift" == 1.4.3
+github "readium/r2-shared-swift" "develop"
```

Comment:
we should probably update this to point to our fork so that we match what we have on SimplyE?

Reply:
yes, should be our repository